### PR TITLE
Feature/tooltip

### PR DESCRIPTION
**Pull request by jaortiz**

### CODE REVIEW

File: components/tooltip/src/Tooltip.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			{children}
+			<span
+				id={tooltipId}
+				css={{
+					visibility: visible ? 'visible' : 'hidden',
+					position: 'absolute',
+					left: '50%',
+					bottom: '110%',
+					transform: 'translate(-50%)',
+					margin: '0 0 6px 0',
```
Comment (jonnystening) [inline components/tooltip/src/Tooltip.js]:
```suggestion
					margin: '0 0 0.375rem 0',
```

File: components/tooltip/src/Tooltip.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+					transitionDelay: '100ms',
+					zIndex: 100,
+
+					'::after': {
+						content: "''",
+						marginLeft: '-0.3125rem',
+						width: 0,
+						borderTop: '5px solid #000',
+						borderRight: '5px solid transparent',
+						borderLeft: '5px solid transparent',
```
Comment (jonnystening) [inline components/tooltip/src/Tooltip.js]:
```suggestion
						borderLeft: '0.3125rem solid transparent',
```

File: components/tooltip/src/Tooltip.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+					transition: 'opacity 0.2 ease, visibility 0.2 ease',
+					transitionDelay: '100ms',
+					zIndex: 100,
+
+					'::after': {
+						content: "''",
+						marginLeft: '-0.3125rem',
+						width: 0,
+						borderTop: '5px solid #000',
+						borderRight: '5px solid transparent',
```
Comment (jonnystening) [inline components/tooltip/src/Tooltip.js]:
```suggestion
						borderRight: '0.3125rem solid transparent',
```

File: components/tooltip/src/Tooltip.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+					pointerEvents: 'none',
+					transition: 'opacity 0.2 ease, visibility 0.2 ease',
+					transitionDelay: '100ms',
+					zIndex: 100,
+
+					'::after': {
+						content: "''",
+						marginLeft: '-0.3125rem',
+						width: 0,
+						borderTop: '5px solid #000',
```
Comment (jonnystening) [inline components/tooltip/src/Tooltip.js]:
```suggestion
						borderTop: '0.3125rem solid #000',
```

**Comment by jaortiz** [comment]
@jonnystening I thought we werent using rems for border widths?

**Pull request merged**